src/ORM/Marshaller.php

```diff
@@ -147,8 +147,8 @@ protected function _buildPropertyMap(array $data, array $options): array
      *   Defaults to true/default.
      * - associated: Associations listed here will be marshalled as well. Defaults to null.
      * - fields: An allowed list of fields to be assigned to the entity. If not present,
-     *   the accessible fields list in the entity will be used. Defaults to null.
-     * - accessibleFields: A list of fields to allow or deny in entity accessible fields. Defaults to null
+     *   the patchable fields list in the entity will be used. Defaults to null.
+     * - patchableFields: A list of fields to allow or deny in entity patchable fields. Defaults to null
```

Comment:
Wondering if we could shorten this to just `patchable`. The `Fields` suffix seems unnecessary.

Reply:
patchable does sound a bit like a boolean flag, though.
But I can understand that the length might be a bit cumbersome.